app/perfunwind.cpp

```diff
@@ -395,6 +395,7 @@ static int frameCallback(Dwfl_Frame *state, void *arg)
     bool isactivation = false;
     dwfl_frame_pc(state, &pc, &isactivation);
     Dwarf_Addr pc_adjusted = pc - (isactivation ? 0 : 1);
+    ui->ipStack.append({pc_adjusted, ui->ipStack.length()});
```

Comment:
move this down to before the `ui->frames.append(frame)` call below, we don't want to append when the cache is dirty e.g.